doc/markup/inline.rst

```diff
@@ -154,6 +154,21 @@ Cross-referencing arbitrary locations
    section headings are changed, and for all builders that support
    cross-references.
 
+   .. versionadded:: 1.4
```

Comment:
I'd have to try myself, but I've got a hunch that you should indent the following paragraphs as to belong to the ``.. versionadded`` directive.